### ISSUE TITLE
Mui.themeProvider has incorrect import statement

### ISSUE THREAD

**Issue by Shmew** [cmeeren/Feliz.MaterialUI#28]
Mui.themeProvider is currently not functioning, as it fails to import.

The generated code is [here](https://github.com/cmeeren/Feliz.MaterialUI/blob/master/src/Feliz.MaterialUI/Mui.fs#L31)

The relevant section of their [export](https://github.com/mui-org/material-ui/blob/master/packages/material-ui/src/index.d.ts) in the index.d.ts file:

```
export {
  createGenerateClassName,
  createMuiTheme,
  createStyles,
  jssPreset,
  makeStyles,
  MuiThemeProvider,
  responsiveFontSizes,
  ServerStyleSheets,
  styled,
  StyleRulesCallback,
  StylesProvider,
  Theme,
  useTheme,
  withStyles,
  WithStyles,
  withTheme,
  WithTheme,
} from './styles';
```

Returns this error when compiling:

```
WARNING in ./src/Client/App/View.fs 251:48-61
"export 'ThemeProvider' was not found in '@material-ui/core'
 @ ./src/Client/Entry.fs
 @ ./src/Client/Client.fsproj
 @ multi ./src/Client/Client.fsproj ./src/Client/scss/main.scss
```

**Comment by cmeeren** [comment]
What? I was certain I tested it. Did they change something again? (Related: #27)

**Comment by cmeeren** [comment]
This comment https://github.com/mui-org/material-ui/issues/17900#issuecomment-543313736 says

> Also, as MuiThemeProvider is no longer documented since v4.0.0, we might as well encourage `ThemeProvider` instead.

Do you know where this `ThemeProvider` is and how to import it?

**Comment by cmeeren** [comment]
Nevermind, I found it: https://material-ui.com/styles/api/#themeprovider

The correct import would be:

```f#
import "ThemeProvider" "@material-ui/core/styles"
```

I'll fix it.

**Comment by cmeeren** [comment]
Will be released shortly as 0.8.1.

**Comment by Shmew** [comment]
Ah, didn't know they were making that change. Thanks!

**Comment by cmeeren** [comment]
Import statement was still broken in 0.8.1, releasing fix in 0.8.2.
